template/config/cypress-ct/cypress.config.js

```diff
@@ -0,0 +1,15 @@
+const { defineConfig } = require('cypress')
+
+module.exports = defineConfig({
+  e2e: {
+    specPattern: 'cypress/e2e/**/*.{cy,spec}.{js,jsx,ts,tsx}',
+    baseUrl: 'http://localhost:4173'
+  },
+  component: {
+    specPattern: 'src/**/__tests__/*.{cy,spec}.{js,ts,jsx,tsx}',
```

Comment:
I chose to explicitly specify `specPatterns` everywhere, for:
1. Make the convention clearer for newcomers;
2. Besides the default pattern, I also allowed `*.spec.*` so that it's easier for users from the old projects to migrate.

Reply:
> Besides the default pattern, I also allowed *.spec.* so that it's easier for users from the old projects to migrate.

I believe this is okay for now.